backend/controllers/clientController.js

```diff
@@ -0,0 +1,59 @@
+const clientModel = require('../models/Client');
+
+exports.getAllClients = (req, res) => {
```

Comment:
Make the function async 

```suggestion
exports.getAllClients = async (req, res) => {
```